AK/Vector.h

```diff
@@ -790,12 +792,12 @@ requires(!IsRvalueReference<T>) class Vector {
     StorageType* inline_buffer()
     {
         static_assert(inline_capacity > 0);
-        return reinterpret_cast<StorageType*>(m_inline_buffer_storage);
+        return m_inline_buffer_storage.address(0);
```

Comment:
calling `address(0)` looks quite awkward to me. Why not just `ptr()` or `base_ptr()`, or something similar?